server/models/pattern/core/pattern.go

```diff
@@ -558,13 +580,14 @@ func createPatternServiceFromK8s(manifest map[string]interface{}) (string, Servi
 	rest = Format.Prettify(rest, false)
 	svc := Service{
 		Name:        name,
-		Type:        w[0].OAMDefinition.Name,
+		Type:        kind,
 		APIVersion:  apiVersion,
 		Namespace:   namespace,
 		Model:       "kubernetes",
```

Comment:
use the modelname retrieved from Component definition.

Reply:
Done